engine/install/index.md

```diff
@@ -73,28 +73,28 @@ and architectures:
 
 > **Note**
 >
-> While the instructions below may work, Docker does not test or verify
-> installation on derivatives.
+> While the instructions below may work, Docker doesn't test or verify
+> installation on distribution derivatives.
 
 - Users of Debian derivatives such as "BunsenLabs Linux", "Kali Linux" or 
   "LMDE" (Debian-based Mint) should follow the installation instructions for
-  [Debian](debian.md), substituting the version of their distro for the
```

Comment:
I must admit I got quite used to `distro` to be used when talking about Linux distributions. For me it helped disambiguate "distribution" (which can have quite some meanings, depending on context), which led to the alternative "Linux distribution" (which is a bit of a mouth full)


I guess I mean to say; do we know how common "distro" is (maybe my view is squashed from within my bubble), and could it still be an alternative to be considered ?

Reply:
@thaJeztah For me it's mostly not using "slang" for people who may not be familiar with it… But in the case of Linux it's probably safe to say they are, so it's not a change I am dedicated to keeping…

Reply:
Yup! That's a fair point, and my view may be skewed indeed. Perhaps we can find out what users would expect to be used for this (so that we can converge on one option)